code/unclassified/src/leap_year/leap_year.java

```diff
@@ -0,0 +1,24 @@
+public class LeapYear {
+	public static boolean isLeapYear(int year) {
```

Comment:
This function is incorrect. Once a year such as 1900, which is not a leap year enters the first branch, the code immediately returns false. I suggest you use a simple return expression:

```java
return (year % 4 == 0 && year % 100 != 0) || (year % 400 == 0);
```